src/clj/backend/main.clj

```diff
@@ -1,21 +1,25 @@
 (ns backend.main
   (:require [aero.core :as aero]
             [backend.routes]
-            [clojure.java.io :as io]
             [clojure.tools.logging :as log]
             [cognitect.transit]
             [hikari-cp.core :as hikari-cp]
             [integrant.core :as ig]
             [next.jdbc.date-time]
             [reitit.ring.middleware.exception]
-            [ring.adapter.jetty :as jetty]))
+            [ring.adapter.jetty9 :as jetty]
+            #_[ring.adapter.jetty :as jetty])
+  (:gen-class))
+
+(set! *warn-on-reflection* true)
```

Comment:
I guess it is good idea to set this in any ns that does any JVM interop. I wonder if there is a way to enable this for the whole project, or is that something we would want.